apps/extension/webpack.config.js

```diff
@@ -0,0 +1,146 @@
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import webpack from 'webpack';
+
+const PRAX = 'lkpmkhpnhknhmibgnmmhdhgdilepfghe';
+
+const IDB_VERSION = 23;
+const USDC_ASSET_ID = 'reum7wQmk/owgvGMWMZn/6RFPV24zIKq3W6In/WwZgg=';
+
+const MINIFRONT_URL = 'https://app.testnet.penumbra.zone/';
+const PENUMBRA_NODE_PD_URL = 'https://grpc.testnet.penumbra.zone/';
+
+const __dirname = new URL('.', import.meta.url).pathname;
+const srcDir = path.join(__dirname, 'src');
+
+export default (env, argv) => {
+  // types declared in prax.d.ts
+  const definitions = {
+    // process.env.NODE_ENV is automatically provided by DefinePlugin
+
+    PRAX: JSON.stringify(PRAX),
+    PRAX_ORIGIN: JSON.stringify(`chrome-extension://${PRAX}`),
```

Comment:
the stringify's feel unnecessary if they are supposed to be strings already

Reply:
defineplugin needs a string to inject https://webpack.js.org/plugins/define-plugin/